packages/server/providers.yaml

```diff
@@ -2,6 +2,7 @@ adobe:
     auth_mode: OAUTH2
     authorization_url: https://ims-na1.adobelogin.com/ims/authorize/v2
     token_url: https://ims-na1.adobelogin.com/ims/token/v3
+    default_scopes: ["offline_access"]
```

Comment:
Thank you for making this an array! 

Let's use this cleaner YAML array notation please:

![Screenshot 2023-04-20 at 11 59 25](https://user-images.githubusercontent.com/2560456/233332239-1a920145-c511-4573-9a88-6c19824b4b79.png)